paper.md

```diff
@@ -35,65 +35,75 @@ bibliography: paper.bib
+While code management<!-- maybe "software development" to use a more intuitive term--> and sharing are streamlined with the advance of software distributions, distributed version control systems, and social coding portals like GitHub, data have remained a “2nd-class citizen” in the contemporary scientific process, despite FAIR principles postulating demands on public data hosting portals and the big callout for Data Science.
+<!-- citation needed, no idea what the big callout is, or how/where FAIR calls for
+public data hosting -->
+Disconnected FAIR<!-- the use of FAIR here confuses, it comes across as a blame--> data hosting portals provide a cacophony of data access and authentication methods, data versioning is frequently ignored, and shared data provenance <!-- what is "shared data provenance"? -->is often not recoverable simply because data management is rarely considered to be an integral part of the scientific process.
+<!-- MIH summary: inconvenient access, version/provenenance information unavailable -->
+DataLad aims to solve these issues by streamlining data consumption, publication, and updating routines, by providing a simplified core API for Git and git-annex operations, and by providing additional features for reproducible science such as command execution <!-- Maybe "execution provenance capture", datalad does not bring "command execution" --> and automatic recomputation based on complete process provenance.
+As it is interoperable with a large variety of scientific and commercial hosting services and available for all operating systems.
+DataLad can be integrated into established systems with minimal adjustments.
+
+## Why Git and git-annex
+
+Git is an excellent <!-- citation needed -->distributed content management system geared towards managing and collaborating on text files <!-- citation needed, does not match self-description -->, but as such it cannot handle large <!-- what is large? --> or binary files well <!-- how come? I have hundreds of repos with binary files in them, works just fine -->.
+Moreover, any data committed to Git becomes available to all clones of that repository. In the context of scientific data, this makes it hard or impossible to provide distributed storage of managed data <!-- why? individual tailoring is hard, but distribution not, what is special about scientific data in this regard? -->, or revoke individual files, such as accidentally stored personal data or data from participants that withdrew from a study.
+Git-annex takes advantage of Git's ability to manage textual information to overcome Git's limitation in managing individual file content that is either large or sensitive.
+File content managed by git-annex is placed into an annex, and in its place, git-annex commits a symbolic link <!-- platform-dependent implementation details, "pointer" could be a term to illustrate the concept and add the "symbolic link" as an example implmentation --> pointing to the content of the file into Git.
```

Comment:
let's replace with "link" since AFAIK it would be the common denominator (to symlink  and git link):
```suggestion
File content managed by git-annex is placed into an annex, and in its place, git-annex commits a link (a symbolic link or a git link) pointing to the content of the file into Git.
```
I do not think it is worth to explain when which, but may be there is a handbook reference we could add?